test_data/test_data.rb

```diff
@@ -0,0 +1,11 @@
+module TestData
+  EXPECTED_APPROXIMATE = 'Approximate round trip times in milli-seconds:'.freeze
+  EXPECTED_AVERAGE = 'Average = 0ms'
+  EXPECTED_DESKTOP = 'Pinging DESKTOP-MRO0O4C'
+  EXPECTED_STATISTICS = 'Ping statistics for ::1:'
+  EXPECTED_HELLO = 'Hello World'
+  TEMP_FILE_PATH = "C:\\test_folder\\temp_file.txt"
+  PING_LOCALHOST = 'ping localhost'
+  REGTEMPLATE = /Minimum = \d+ms, Maximum = \d+ms, Average = \d+ms/
+  OUTWRITEPLUS = 'w+'
```

Comment:
for grouped data we can use hashes
eg 
       
EXPECTED_DATA = {
     DESKTOP = 'Pinging DESKTOP-MRO0O4C',
     HELLO = 'Hello World'
}

Reply:
Done.